product_service/lambda_func/products.ts

```diff
@@ -0,0 +1,38 @@
+export const products = [
```

Comment:
I recommend to define a type for this const

```suggestion
export const products: Array<Product> = [
```